Rakefile

```diff
@@ -25,6 +22,16 @@ task :build do
   system 'gem build cequel.gemspec'
 end
 
+desc 'Build the package and publish it to rubygems.pkg.github.com'
+task publish: :build do
+  require 'cequel'
```

Comment:
```suggestion
```

Reply:
Why is this unnecessary? it was included for the jsonAPI change that you linked

Reply:
Sorry if I shared a stale link. Here's a fresh one: https://github.com/art19/jsonapi.rb/pull/5/files.

You can run the command locally to confirm, you can update

```
  system("gem push --key github --host https://rubygems.pkg.github.com/art19 " \
```

to something like 

```
  system("echo gem push --key github --host https://rubygems.pkg.github.com/art19 " \
```

to play with it locally.

Reply:
this is my bad I must have hallucinated this line